staging/https-nginx/README.md

```diff
@@ -6,10 +6,10 @@ It uses an [nginx server block](http://wiki.nginx.org/ServerBlockExample) to ser
 
 ### Generate certificates
 
-First generate a self signed rsa key and certificate that the server can use for TLS. This step invokes the make_secret.go script in the same directory, which uses the kubernetes api to generate a secret json config in /tmp/secret.json.
+First generate a self signed rsa key and certificate that the server can use for TLS.
 
 ```sh
-$ make keys secret KEY=/tmp/nginx.key CERT=/tmp/nginx.crt SECRET=/tmp/secret.json
```

Comment:
do you perhaps need to remove the `keys` target from the makefile now?

Reply:
you are right, I pushed a new commit that improves my change